tests/test_sphinx/test_sphinx_builds/test_bibtex_norefs.xml

```diff
@@ -0,0 +1,7 @@
+<document source="index.md">
+    <paragraph>
+        missing ref 
+        <pending_xref classes="bibtex" ids="id1" refdomain="citation" reftarget="abc" reftype="ref" refwarn="True" support_smartquotes="False">
+            <inline>
+                [abc]
+    <paragraph ids="bibtex-bibliography-index-0">
```

Comment:
@choldgraf, re formatting issues, perhaps it's because the final paragraph does not have any content.